webview/src/setup/components/remote/Connect.tsx

```diff
@@ -18,5 +20,6 @@ export const Connect: React.FC = () => (
       </a>{' '}
       for details on how to connect to a remote
     </p>
+    <StartButton onClick={() => addRemote()} text="Add Remote" />
```

Comment:
Since you can have multiple remotes in a project, should we keep showing the "Add Remote" button when a user has already added remotes?

Reply:
Actually, now that I think about it more, the goal of this section is to setup remotes, not manage remotes, so it does make sense for the button to disappear.

Reply:
Discussed this today. For anyone else finding this. Adding the command is in the works but the reasons for leaving out the button from the remotes connected page in this PR are as follows:

1. Would be strange to only have a single option to add remotes when you have already set at least one up (need to add more actions to make this seem reasonable)
2. PR is big enough